frigate/ffmpeg_presets.py

```diff
@@ -32,7 +32,7 @@ def get_selected_gpu(self) -> str:
         devices = list(filter(lambda d: d.startswith("render"), os.listdir("/dev/dri")))
 
         if len(devices) < 2:
-            self._selected_gpu = devices[0]
+            self._selected_gpu = f"/dev/dri/{devices[0]}"
```

Comment:
Functionally it would be a potentially minor change, but is there a benefit to simply removing this entire if statement for early return and using the following for loop that will do the same thing AFTER verifying the device?

Reply:
if there is only one better to just try it in ffmpeg as opposed to relying on vainfo driver check to return success